src/components/SearchPage/SearchPageComponent.js

```diff
@@ -64,39 +46,29 @@ export function SearchPageComponent() {
     dispatch(searchResultClickedAction({ note }));
   };
 
-  const classes = useStyles();
+  const responseList =
```

Comment:
Why `responseList` if it's JSX for search results? :)
Since you have renamed `searchResults` into `resultList`, maybe you can name this variable `resultListElement`.
Or make it a function `renderResultList()`.